### PR TITLE
Revamp media player

### LINKED ISSUE #8540: Remove all direct API calls from frontend to middleware
**Describe the bug**

In the new middleware version, we can't call middleware directly from frontend

currently effected: dev-middleware only and staging middleware is working well as it is using production middleware version

**To Reproduce**

**Expected behavior**

- [ ] change all the usage in frontend to support current middleware version

**Screenshots**

### PR DESCRIPTION
**Pull request by sainak**
## Proposed Changes

- Replace hls player with mss on safari
- remove hls related code
- remove old unused components

https://developer.apple.com/support/app-store/
https://caniuse.com/mdn-api_managedmediasource_streaming

closes https://github.com/ohcnetwork/care_fe/issues/8540

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Review by rithviknishad — APPROVED** [review]

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
@sainak 
image

- [ ] feed is not working on safari nor on chrome in iPhone version <18.0

CC: @gigincg @vigneshhari

**Comment by nihal467** [comment]
video is visible is all devices now LGTM

File: src/Components/CameraFeed/videoPlayer.tsx [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+      playerRef.current.onplaying = () => {
+        props.onSuccess?.(undefined);
+      };
+    } catch (err) {
+      console.debug(err);
+      props.onError?.(err);
+    }
+  };
+
+  const startmse = () => {
```
Comment (rithviknishad) [inline src/Components/CameraFeed/videoPlayer.tsx]: https://docs.aws.amazon.com/prescriptive-guidance/latest/best-practices-cdk-typescript-iac/typescript-best-practices.html#naming-conventions

```suggestion
  const startMSE = () => {
```

File: src/Components/CameraFeed/videoPlayer.tsx [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+      mseSourceBuffer.abort();
+    }
+    if (playerRef.current) {
+      playerRef.current.pause();
+      playerRef.current.src = "";
+      playerRef.current.srcObject = null;
+    }
+  };
+
+  const starthls = () => {
```
Comment (rithviknishad) [inline src/Components/CameraFeed/videoPlayer.tsx]:
```suggestion
  const startHLS = () => {
```

File: src/Components/CameraFeed/FeedAlert.tsx [hunk 3]
```diff
@@ -2,14 +2,15 @@ import { Transition } from "@headlessui/react";
 import { useEffect, useState } from "react";
 import CareIcon, { IconName } from "../../CAREUI/icons/CareIcon";
 import { classNames } from "../../Utils/utils";
-import { StreamStatus } from "./usePlayer";
+export type StreamStatus = "playing" | "stop" | "loading" | "offline";
```
Comment (rithviknishad) [inline src/Components/CameraFeed/FeedAlert.tsx:5]: Duplicate declaration of `StreamStatus`

File: src/Components/CameraFeed/FeedNetworkSignal.tsx [hunk 4]
```diff
@@ -1,7 +1,8 @@
 import { useEffect, useState } from "react";
 import { calculateVideoDelay } from "./utils";
 import NetworkSignal from "../../CAREUI/display/NetworkSignal";
-import { StreamStatus } from "./usePlayer";
+
+export type StreamStatus = "playing" | "stop" | "loading" | "offline";
```
Comment (rithviknishad) [inline src/Components/CameraFeed/FeedNetworkSignal.tsx]: Duplicate declaration of `StreamStatus`

**Review by rithviknishad — APPROVED** [review]

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**